src/somef/cli.py

```diff
@@ -786,7 +809,7 @@ def load_local_repository_metadata(local_repo):
         with open(os.path.join(os.path.join(repo_dir, "README.MD")), "r", encoding='utf-8') as data_file:
             text = data_file.read()
     elif os.path.exists(os.path.join(repo_dir, "README.md")):
-        with open(os.path.join(os.path.join(repo_dir, "README.md")), "r", encoding='utf-8') as data_file:
+        with open(os.path.join(os.path.join(repo_dir, "README.MD")), "r", encoding='utf-8') as data_file:
```

Comment:
This is problematic. It's going to fail in unix because you are checking that README.md exists but then opening README.MD